report_py3o/models/py3o_report.py

```diff
@@ -302,6 +305,23 @@ def _zip_results(self, reports_path):
                 cpt += 1
         return result_path
 
+    @api.model
+    def _merge_pdf(self, reports_path):
+        """ Merge PDF files into one.
+
+        :param reports_path: list of path of pdf files
+        :returns: path of the merged pdf
+        """
+        writer = PdfFileWriter()
```

Comment:
Hi. Thanks for your work!
if you did not know: there is a function in odoo core for pdf merge: https://github.com/odoo/odoo/blob/12.0/odoo/tools/pdf.py#L7
However it expects a pdf file and not a path. Feel free to include it or leave it as is. :+1:

Reply:
@BT-kaberer Thank you for the reference, I'm aware of this method but I don't like it since it requires to have all the documents into memory. If you take a look at the implementation of py3o_report you'll see that we try to minimize the memory footprint of the generation process by working with files and streams ... https://github.com/OCA/reporting-engine/pull/258/files#diff-718f50f1e1aa99232a24143065702955R376